.github/rust.yml

```diff
@@ -0,0 +1,23 @@
+name: Rust
+
+on:
+  push:
+    branches: [ '*' ]
+  pull_request:
+    branches: [ master, withdraw_from_anchor_if_negative_profit, run_unit_tests ]
```

Comment:
withdraw_from_anchor_if_negative_profit, run_unit_tests is a temporary branches, right?
maybe we should leave only one branch on PR - master?

or, for example, do something like this
```
on:
  push:
    branches-ignore: 
      - "master"
  pull_request:
    branches: 
      - "master"
```

Reply:
so I was thinking its best to run this on most situations to catch errors when they occur

Reply:
yep this configuration allows to run tests in any other branches on push event except master, and run tests only on PR on master branch